lib/ansible/modules/cloud/amazon/elasticache.py

```diff
@@ -526,13 +526,17 @@ def main():
     cache_parameter_group = module.params['cache_parameter_group']
 
     if cache_subnet_group and cache_security_groups:
-        module.fail_json(msg="Can't specify both cache_subnet_group and cache_security_groups")
+        module.fail_json(
+            msg="Can't specify both cache_subnet_group and cache_security_groups")
```

Comment:
Don't need to add the line break here. Line length needs to be less than 160 characters and this is well within.